src/fileSyncer.ts

```diff
@@ -91,11 +91,14 @@ export class FileSyncer {
             }
             if (fileName != null) {
                 // check if file under configured source path
+                if (this.isExcluded(fileName)) {
```

Comment:
it's more clear to have 2 layer check. if null, then still need to go to next block to copy whole folder. if not null and not isExcluded, loop to next interate. otherwise, it will be if ..elif... else